src/helpers.js

```diff
@@ -15,7 +15,10 @@ const {
   FILE_TYPE,
   INPUT_COURSE_DATE_FORMAT,
   YOUTUBE_SHORTCODE_PLACEHOLDER_CLASS,
-  ROOT_RELATIVE_REGEX
+  ROOT_RELATIVE_REGEX,
```

Comment:
This and a couple other constants here aren't used anymore.  Can we remove unused imports?